internal/utils/ocpbuild/helper.go

```diff
@@ -13,12 +13,12 @@ import (
 
 var ErrNoMatchingBuild = errors.New("no matching Build")
 
-//go:generate mockgen -source=helper.go -package=build -destination=mock_helper.go
+//go:generate mockgen -source=helper.go -package=ocpbuild -destination=mock_helper.go
 
 type OpenShiftBuildsHelper interface {
-	GetModuleBuildByKernel(ctx context.Context, mld *api.ModuleLoaderData) (*buildv1.Build, error)
-	GetModuleBuilds(ctx context.Context, moduleName, moduleNamespace string) ([]buildv1.Build, error)
-	DeleteBuild(ctx context.Context, build *buildv1.Build) error
+	GetModuleOCPBuildByKernel(ctx context.Context, mld *api.ModuleLoaderData) (*buildv1.Build, error)
+	GetModuleOCPBuilds(ctx context.Context, moduleName, moduleNamespace string) ([]buildv1.Build, error)
+	DeleteOCPBuild(ctx context.Context, build *buildv1.Build) error
```

Comment:
Should we remove the `OpenShift` prefix from the object? How about removing `OCP` from the methods?
We can always name the filed holding that object `ocpBuildHelper`. Do we also want it in the object/methods names?

Reply:
same answer as above